### DISCUSSION TITLE
azureAssistants and using Retrieval Capabilities creating the Assistant via the interface is not possible.

### DISCUSSION THREAD

**Discussion by ervet** [danny-avila/LibreChat#4432] [Troubleshooting] [answered]
### What happened?

Creating an assistant via the LibreChat Assistant Builder (azureAssistants) and mark the retrieval Capability is not possible. Saving causes and error and the assistant is not created.

In the Debug Log the following appears. 

error: [/assistants] Error creating assistant 400 Invalid value: 'retrieval'. Supported values are: 'code_interpreter', 'function', 'file_search', and 'opena... [truncated]

How to fix this issue. 

### Steps to Reproduce

Creating the Assistant with Retrieval enabled

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
It's not supported with Azure Assistants v1, which LC is setup to use. This requires V2, which is not yet implemented. There is currently no ETA as this is not a priority for the project.

**Reply by rio100** [reply]
I agree with that.

-Jerry
